### ISSUE TITLE
High Throughput Algae Cell Detection

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#417]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : High Throughput Algae Cell Detection</br>
:red_circle: **Aim** : The aim of this project is to detect high throughput Algae cell using Deep Learning and OpenCV methods.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/marquis03/high-throughput-algae-cell-detection</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by ARPIT2128** [comment]
## 👋 Hello!

### Participant Details:
- **Full Name:** Arpit Sharma
- **GitHub Profile:** [Arpit2128](https://github.com/ARPIT2128)
- **Email ID:** arpitsharma2128@gmail.com

### Approach for the Project:
I'm planning to explore the following models for the project:
- [Xception](https://keras.io/api/applications/xception)
- [ConvNeXtTiny](https://keras.io/api/applications/convnext/#convnexttiny-function)
- [InceptionV3](https://keras.io/api/applications/inceptionv3)
- [YOLOv8m](https://github.com/ultralytics/assets/releases/download/v0.0.0/yolov8m.pt)

### Reason for Choosing These Models:
- All mentioned models have approximately close parameters, making them suitable for a comprehensive comparison.
- I have previous experience working with both pre-trained CNN architectures and YOLOv8 base architectures (tiny and base) for projects like [face-mask-detection](https://github.com/ARPIT2128/SAP-internal-face-mask-detection) and more.

### Dataset and Models Used:
- I'll be using the [dataset](https://www.kaggle.com/datasets/marquis03/high-throughput-algae-cell-detection) provided in the project.
- One of the approaches i recently went through was using [YOLOV5](https://www.kaggle.com/code/marquis03/yolov5-high-throughput-algae-cell-detection) for this dataset (YOLOV5 links to the notebook mentioning the approach).

### Learning and Improvement Goals:
- I'm particularly keen on improving my pre-processing strategies throughout this project.
- Improving git and open source knowledge by contributing and interacting with Project Admins.
- Strengthening my DL foundations. 

### Participant Role:
- **Open Source Program:** SWOC 2024
- **Role:** Contributor

Looking forward to collaborating and contributing to the success of the project! 🚀
****

**Comment by abhisheks008** [comment]
Issue assigned to you @ARPIT2128
